src/KafkaFlow.Admin/Handlers/ConsumerMetricHandler.cs

```diff
@@ -0,0 +1,19 @@
+namespace KafkaFlow.Admin.Handlers
+{
+    using System.Threading.Tasks;
+    using KafkaFlow.Admin.Messages;
+    using KafkaFlow.TypedHandler;
+
+    internal class ConsumerMetricHandler : IMessageHandler<ConsumerMetric>
+    {
+        private readonly ITelemetryCache cache;
```

Comment:
Changing the ITelemetryCache interface name, we need to change the names of the fields too.